lib/pry/commands/play_motion_picture.rb

```diff
@@ -0,0 +1,125 @@
+    ['https://www.youtube.com/watch?v=Z_ut1L7NH30', 'In the name of the Father - Giuseppe is dead,
+                                                    man.'],
+    ['https://www.youtube.com/watch?v=NoOhnrjdYOc', 'English Rose - Elton John.'],
+    ['https://www.youtube.com/watch?v=Xu3FTEmN-eg', 'The Chemical Brothers - Galvanize'],
+    ['https://www.youtube.com/watch?v=gCYcHz2k5x0', 'Martin Garrix - Animals'],
+    ['https://www.youtube.com/watch?v=kn15BUFBvu8', 'Fa-fa-fa-fire! - Fawlty Towers'],
+    ['https://www.youtube.com/watch?v=qEYueRVuqmg', 'Tiesto - Just Be'],
+    ['https://www.youtube.com/watch?v=WlBiLNN1NhQ', "Always Look on the Bright Side of Life -
+                                                     Monty Python's Life of Brian"],
+    ['https://www.youtube.com/watch?v=7gMJBQoHJ4E', 'Billy Connolly - Terrorist Attack
+                                                    At Glasgow Airport'],
+    ['https://www.youtube.com/watch?v=JqZo07Ot-uA', 'Billy Connolly - Algebra'],
+    ['https://www.youtube.com/watch?v=zs8QKXtCN9w', 'The Mad Irishman - Braveheart'],
+    ['https://www.youtube.com/watch?v=UzWHE32IxUc', 'Lenny Kravitz - American Woman'],
+    ['https://www.youtube.com/watch?v=hIvRkjOd1f8', 'Braveheart: Freedom Speech'],
+    ['https://www.youtube.com/watch?v=IcrbM1l_BoI', 'Avicii - Wake Me Up'],
+    ['https://www.youtube.com/watch?v=9jE9bXvX5cM', '11- The Beatles - Cold turkey'],
+    ['https://www.youtube.com/watch?v=vefJAtG-ZKI', 'The Beatles Yellow Submarine'],
+    ['https://www.youtube.com/watch?v=e-7nbcWOaws', 'Potatoes Of The Night - Billy Connolly'],
+    ['https://www.youtube.com/watch?v=cbB3iGRHtqA', 'Scooter - How Much Is The Fish?'],
+    ['https://www.youtube.com/watch?v=X-idP23bHCg', 'A gift of Love'],
+    ['https://www.youtube.com/watch?v=viaTT859Yk0', 'Sifl & Olly - United States of Whatever'],
+    ['https://www.youtube.com/watch?v=YVkUvmDQ3HY', 'Eminem - Without Me'],
+    ['https://www.youtube.com/watch?v=TLV4_xaYynY', 'The Jimi Hendrix Experience ' \
+                                                    '- All Along The Watchtower'],
+    ['https://www.youtube.com/watch?v=YTBC7ckTWpo', 'Luke Kelly - Scorn Not His Simplicity'],
+    ['http://sayyidali.com/leading-iran/o-leader-please-dont-talk-about-leaving.html',
+     'free to disagree, doesn\'t make us enemies']
+  ].freeze
+
+  MOTION_PICTURES_SEQ = MOTION_PICTURES.dup.to_enum
+
+  match 'play-motion-picture'
+  group 'Misc'
+  description 'Bored? Play a motion picture.'
+  command_options argument_required: false
+
+  def options(o)
+    o.on :m,
+     :match,
+     'Play first motion picture that matches through a fuzzy zoeke (search)',
+     argument: true
+    o.on :s,
+         :sequential,
+         'Play motion pictures in sequential order, until the end.',
+         argument: false
+  end
+
+  def process
+    case
+    when opts[:sequential]
+      begin
+        play MOTION_PICTURES_SEQ.next[0]
+      rescue Pry::RescuableException
```

Comment:
@banister https://github.com/pry/pry/pull/1639#discussion_r137670864 
is this the main issue with merging that pull request, or is there more to think about?
I would like to move towards `Pry.Rescuable()` instead.